src/crimson/os/seastore/seastore_types.h

```diff
@@ -212,6 +232,8 @@ enum class extent_types_t : uint8_t {
   LADDR_INTERNAL = 1,
   LADDR_LEAF = 2,
   ONODE_BLOCK = 3,
+  EXTMAP_INNER = 4,
+  EXTMAP_LEAF = 5,
```

Comment:
Looks the implementation in `Cache::get_extent_by_type()` is missing.

Reply:
Yeah, you do need to add that for any new extent types.

Reply:
added in Cache::get_extent_by_type(). updated.